node-cli/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "prompto",
+  "version": "1.0.0",
+  "description": "",
+  "main": "prompt.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "babel-core": "^6.26.3",
+    "chalk": "^4.1.0",
+    "clear": "^0.1.0",
+    "events": "^3.1.0",
+    "figlet": "^1.5.0",
+    "inquirer": "^7.3.2",
+    "mongodb": "^3.5.9",
```

Comment:
It seems to me that you're only depending on this for the bson types.
It might be simpler to depend directly on the "bson" package?